pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -135,8 +138,10 @@ class Build {
             suffix="corretto/corretto-${javaNumber}"
         } else if (buildConfig.VARIANT == "openj9") {
```

Comment:
Hmm we need to refactor this overloaded term VARIANT should probably refer to the underlying VM and we should introduce PROVIDER for 'who is behind this thing'

Reply:
agreed, we can do some logic at the publish phase to control this for now.

Reply:
PR to push these binaries to a separate repo for the API: https://github.com/AdoptOpenJDK/openjdk-website-backend/pull/72/files